keptn-gateway/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+
+	"fmt"
+	"log"
+	"net/http"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	metrics_v1 "github.com/keptn/lifecycle-toolkit/metrics-operator/api/v1"
+)
+
+var clientset *kubernetes.Clientset
+
+func handlerMetricsCount(w http.ResponseWriter, r *http.Request) {
+	log.Println("http call to /pods")
```

Comment:
we should use a better logging framework, for example: https://github.com/uber-go/zap/tree/master